eng/scripts/Language-Settings.ps1

```diff
@@ -128,4 +128,47 @@ function Get-dotnet-GithubIoDocIndex() {
   $tocContent = Get-TocMapping -metadata $metadata -artifacts $artifacts
   # Generate yml/md toc files and build site.
   GenerateDocfxTocContent -tocContent $tocContent -lang "NET"
+}
+
+# details on CSV schema can be found here
+# https://review.docs.microsoft.com/en-us/help/onboard/admin/reference/dotnet/documenting-nuget?branch=master#set-up-the-ci-job
+function Update-dotnet-CIConfig($pkgs, $ciRepo, $locationInDocRepo, $monikerId=$null){
```

Comment:
Aha. because $monikerId isn't strictly necessary for dotnet, but want to keep the call identical. 👍